File: python/5.py
#!/usr/bin/env python
step_size = 20
target = step_size
numbers_to_attempt = range(target, 0, -1)

while True:
    success = True
    for divisor in numbers_to_attempt:
        if target % divisor != 0:
            success = False
            break
    if success:
        print(target)
        break
    target += step_size

File: rust/src/bin/5.rs
fn main() {
    let step_size: i32 = 20;
    let mut target = step_size;

    loop {
        let mut success: bool = true;
        for i in 1..step_size {
            if target % i != 0 {
                success=false;
                break;
            }
        }
        if success == true {
            println!("{}", target);
            break;
        }
        target += step_size;
    }
}
